### PR TITLE
Need a combination for ASCAT and allelecounter for nf-core

### PR DESCRIPTION
**Pull request by lassefolkersen**

### CODE REVIEW

**Comment by lassefolkersen** [comment]
Hi @bgruening -- I wanted to make this in the web-interface, but unfortunately the ASCAT v3.0.0 is not there yet (probably because it was released yesterday 😄 )

**Comment by bgruening** [comment]
Mh I see them here: https://quay.io/repository/biocontainers/ascat?tag=latest&tab=tags what are you missing? :)

**Comment by lassefolkersen** [comment]
I need to have the two containers ([this](quay.io/biocontainers/ascat:3.0.0--r41hdfd78af_0) and [this](quay.io/wtsicgp/allelecount:v4.3.0)) combined, and the good people at nf-core told me this was the place to do it. So I made a pull request (this one). What I wrote about web-interface, that was just for [this helper service](https://biocontainers.pro/multipackage) where ASCAT didn't show in the 3.0.0 version (still doesn't). So I though I could request the mulling by a pull request instead. @bgruening

**Comment by lassefolkersen** [comment]
But I can see it's failing now... I'm a little unsure why. Some warning on not hard-coding URLs. I guess I could just write cancerit-allelecount=4.3.0 and ascat=3.0.0 they are both on bioconda. They are just needed together, to be run as an nf-core module in sarek.

File: combinations/hash.tsv [hunk 1]
```diff
@@ -194,3 +194,4 @@ python=3.9.5,samtools=1.14
 bioconductor-diffbind=3.4.0,bioconductor-csaw=1.28.0,bioconductor-edger=3.36.0,r-optparse=1.7.1
 r-shazam=1.1.0,r-tigger=1.0.0
 bamcmp=2.2	quay.io/bioconda/base-glibc-debian-bash:latest	0
+ascat=3.0.0 cancerit-allelecount=4.3.0
```
Comment (bgruening) [inline combinations/hash.tsv:197]: just look above, you need to separate them via comma :)

**Comment by lassefolkersen** [comment]
Thanks @bgruening ! 😄

**Pull request merged**